contracts/SimpleNftMarketplace.sol

```diff
@@ -60,25 +60,30 @@ contract SimpleNftMarketplace is ListingManager, ValidateSignature {
 
   // Admin
   function changeSupportedContract(address contractAddress, bool isSupported) external onlyAdmin returns (bool success) {
-    return false;
+    _changeSupportedContract(contractAddress);
+    return true;
   }
 
-  function changeTransactionFee(uint32 transactionFee) external onlyAdmin returns (bool success) {
-    return false;
+  function changeTransactionFee(uint32 newTransactionFee) external onlyAdmin returns (bool success) {
+    _changeTransactionFee(newTransactionFee);
+    return true;
   }
 
   // Treasury
   function withdrawTransactionFee() external onlyTreasury returns (bool success) {
-    return false;
+    _withdrawTransactionFee();
+    return true;
   }
 
   // Moderator
-  function blacklistToken(address tokenContract, uint256 tokenId) external onlyModerator returns (bool success) {
-    return false;
+  function blacklistToken(address tokenContract, uint256 tokenId, bool isBlackListed) external onlyModerator returns (bool success) {
+    _blacklistToken(tokenContract, tokenId, isBlackListed);
+    return true;
```

Comment:
_blacklistToken(tokenContract, tokenId, isBlackListed) already return bool
so change for
```solidity
return _blacklistToken(tokenContract, tokenId, isBlackListed);
```